src/PABLO/LocalTree.cpp

```diff
@@ -906,10 +906,9 @@ namespace bitpit {
 
         // If a face is a boundary, it can have neighbours only if periodic
         if (oct->m_info[iface]){
-            // Use the proper function for searching periodic neighbours
+            // Set searching periodic neighbours
             if (m_periodic[iface]){
                 isperiodic = true;
-//                findPeriodicNeighbours(oct, iface, neighbours, isghost);
             }
```

Comment:
These changes belong to the previous commit.

Reply:
Yes, right.